connectors/sources/jira.py

```diff
@@ -516,6 +516,7 @@ def get_default_configuration(cls):
                 "label": "Jira Cloud service account id",
                 "order": 6,
                 "type": "str",
+                "tooltip": "Email address associated with Jira Cloud account. E.g. alex.wilber@gmail.com",
```

Comment:
```suggestion
                "tooltip": "Email address associated with Jira Cloud account. E.g. jane.doe@gmail.com",
```

I'd use generic name/surname here.

Also how do you feel changing the label too? Why does it have "id" in label btw?

Reply:
Yes, i've kept it to `email address` to avoid confusion